tests/integration/cri-containerd/gha-run.sh

```diff
@@ -14,7 +14,48 @@ cri_containerd_dir="$(dirname "$(readlink -f "$0")")"
 source "${cri_containerd_dir}/../../common.bash"
 
 function install_dependencies() {
-	return 0
+	info "Installing the dependencies needed for running the cri-containerd tests"
+
+	# Dependency list of projects that we can rely on the system packages
+	# - build-essential
+	#   - Theoretically we only need `make`, but doesn't hurt to install
+	#     the whole build-essential group
+	# - jq
+	# - podman-docker
+	#   - one of the tests rely on docker to pull an image.
+	#     we've decided to go for podman, instead, as it does *not* bring
+	#     containerd as a dependency
+	declare -a system_deps=(
+		build-essential
+		jq
+		podman-docker
```

Comment:
I'm not sure that podman-docker is available on ubuntu 20:04, so I don't know if you anticipate this running on that environment ever?

Reply:
Maybe we will, and then we can figure out what to do when the time comes.

I think we have the freedom to configure the external runners to using a recent enough version of Ubuntu.  If I'm  proven wrong, I'll make sure to help whoever hits this issue.